scania-digital-design-system/tegel
# fix(table-component-expandable-rows): format args for default value

## **Describe pull-request**  
This pull request aims to fix the production issue we have with storybook page for table expandable row. The arguments are not defaulted due to the `args` is not within the correct brackets.

## **How to test**  
Provide detailed steps for testing, including any necessary setup._
Go to the preview link and navigate to the table expandable row page.
The controls should be set to default values.

## **Checklist before submission**

## **Suggested test steps**

## **Screenshots**  
_Include before/after screenshots for UI changes._

## **Additional context**  
_Add any other context or feedback requests about the pull-request here._
